src/scripts/utilities.js

```diff
@@ -251,4 +266,52 @@ Results in the bot creating a study session and responding with a message that l
 	});
 }
 
+function handleHelpRoleMessageCommand(message) {
+	message.channel.send(null, {
+		embed: {
+			title: "The !role message command",
+			fields: [
+				{
+					name: 'Description',
+					value: 'The `!role message` command can be used to create self-assigned role selection menus'
+				}, {
+					name: 'Format',
+					value: `
+The role message command requires
+- a title placed on the first line (beside the \`!role message\` command)
+- at least one role option in the format: \`emoji: roleId - description\`
+
+Each role goes on its own line
+The emojis should be from the standard set (no custom emojis)
+The description has a 50 character limit
+
+Besides these rules, the message can have any additional information or formatting added
+					`
+				}, {
+					name: 'Example',
+					value: `
+Sending a message like this:
+\`\`\`
+!role message Alphabet roles
+
+Here's some information
+
+:boom:: 819317488923049987 - For people who want to have role A
+This role gives you access to #test-a
+:knife:: 819317563841314816 - For people who want to have role B
+This role gives you access to #test-b
+
+Here's some more information. Here's \_some\_ \*\*formatting\*\*
+\`\`\`
+Results in the bot responding with a message that looks like this:
+					`
+				}
+			],
+			image: {
+				url: 'https://i.imgur.com/Mjp3pVU.png'
+			}
+		}
+	});
+}
+
```

Comment:
Message content is super long, maybe you can pull it in a new CONSTANT file 
https://github.com/chris-mcdonald-dev/korean-discord-app/tree/development/src/constants